File: CoreJavaAssignment/CoreJavaAssignment2/Tshirt.java
package CoreJavaAssignment2;

public class Tshirt {
	String color;
	String material;
	String design;
	Tshirt(String col,String mater,String deg)
	{
		color=col;
		material=mater;
		design=deg;
	}
	void display()
	{
	System.out.println(color +" "+material+" "+design);
	
	}
	 public String Size(String size)
	{
		return size;
	}
	 public static void main(String args[])
	 {
		 Tshirt t1=new Tshirt("blue","fabric","prited");
		 t1.display();
		 System.out.println(t1.Size("small"));
		 
		 Tshirt t2=new Tshirt("red","woolen","textile");
		 t2.display();
		 System.out.println(t2.Size("large"));
		 Tshirt t3=new Tshirt("black","cotton","fasion");
		 t3.display();
		 System.out.println(t3.Size("extra large"));
		 
	 }
} 

File: typescriptAssignment/tshirt.ts
class Tshirt{
    color:string;
    material:string;
    design:string;
    constructor(c:string,m:string,d:string)
    {
         this.color=c;
         this.material=m;
         this.design=d;
    }
setsize(size:string)
{
   return size;
}
  display()
  {
      console.log(this.color);
      console.log(this.material);
      console.log(this.design);
  }
    
}
var t1 = new Tshirt("pink","linen","abstract");
t1.display();
console.log(t1.setsize("Small"));